app/tests/src/test_citations.py

```diff
@@ -17,9 +17,14 @@ def test_get_context_for_prompt():
 
 
 def test_add_citations():
-    assert add_citations("This is a citation (chunk-0)", []) == "This is a citation (chunk-0)</br>"
+    assert add_citations("This is a citation (chunk-0)", []) == "This is a citation (chunk-0)"
 
     chunks = ChunkFactory.build_batch(2)
+    assert (
+        add_citations("This is a citation (chunk-0) and another (chunk-1).", chunks)
+        == "This is a citation <sup><a href='#'>1</a>&nbsp;</sup> and another <sup><a href='#'>2</a>&nbsp;</sup>."
```

Comment:
Trying to understand `<a href='#'>`: should there be an anchor name after `#`?
Found somewhat related https://stackoverflow.com/questions/37640583/understanding-a-href

Reply:
It's a placeholder from this line: https://github.com/navapbc/labs-decision-support-tool/blob/kb/improve-citation-ux/app/src/citations.py#L42

We could also update the code to simply not wrap non-BEM citations in links if you prefer!

Reply:
I'm not following, but testing it locally, it works!

Reply:
Oh, wait. It's b/c `chunk.document.name` is not present.